host/tests/L3_TestCases/dsVideoDevice/dsVideoDevice_test1_FrameratePreChangePostChangeCallback_Verify.py

```diff
@@ -127,5 +128,7 @@ def testFunction(self):
         return result
 
 if __name__ == '__main__':
-    test = dsVideoDevice_test1_FrameratePreChangeCallback_Verify()
+    summerLogName = os.path.splitext(os.path.basename(__file__))[0] + "_summery"
+    summeryLog = logModule(summerLogName, level=logModule.INFO)
+    test = dsVideoDevice_test1_FrameratePreChangePostChangeCallback_Verify()
```

Comment:
Please pass the summaryLog object to test object. Currently the summaryLog Object is not used anywhere.
Please check all the tests.

Reply:
Updated in all the tests and tested it.